manifests/extension.pp

```diff
@@ -74,6 +78,7 @@
   Optional[String] $ini_prefix                      = undef,
   Optional[String] $php_api_version                 = undef,
   String           $package_prefix                  = $php::package_prefix,
+  Optional[String] $package_name                    = undef,
```

Comment:
please enforce the minimal string length:
```suggestion
  Optional[String[1]] $package_name                 = undef,
```

Reply:
Added